internal/integration/helpers.go

```diff
@@ -319,3 +375,63 @@ func Eventually(tb testing.TB, times uint64, f func() error) {
 func IndexFilePath() string {
 	return path.Join(FileNameRoot, "index.txt")
 }
+
+func newSigningKey(tb testing.TB) *signingKey {
+	tb.Helper()
+	privateKey, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+	if err != nil {
+		tb.Fatal(err)
+	}
+
+	publicKey := privateKey.Public()
+	x509EncodedPub, err := x509.MarshalPKIXPublicKey(publicKey)
+	if err != nil {
+		tb.Fatal(err)
+	}
+	pemEncodedPub := pem.EncodeToMemory(&pem.Block{Type: "PUBLIC KEY", Bytes: x509EncodedPub})
+	pemPublicKey := string(pemEncodedPub)
+
+	return &signingKey{
+		Key:       privateKey,
+		PublicKey: pemPublicKey,
+	}
+}
+
+// Based on the publish request, generate a JWT as if it came from the
+// authorized health authority.
+func issueJWT(t *testing.T, cfg jwtConfig) (jwtText, hmacKey string) {
```

Comment:
I Think this came from another test - can we put it in a centrally located place and have them both depend on it?

can't be here because I think that would be a circular dependency

Reply:
Moved it to the verification util lib